packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import $ from '@js/core/renderer';
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+
+import Widget from '../widget';
+
+const CHAT_MESSAGE_AVATAR_CLASS = 'dx-chat-message-avatar';
+const CHAT_MESSAGE_AVATAR_LETTERS_CLASS = 'dx-chat-message-avatar-letters';
+
+export interface AvatarOptions extends WidgetOptions<Avatar> {
+  name?: string;
+}
+
+class Avatar extends Widget<AvatarOptions> {
+  _getDefaultOptions(): AvatarOptions {
+    return {
+      ...super._getDefaultOptions(),
+      ...{
+        name: '',
+      },
+    };
+  }
+
+  _getAvatarInitials(name: string): string {
+    const initials = `${name.charAt(0).toUpperCase()}`;
+
+    return initials;
+  }
+
+  _initMarkup(): void {
+    $(this.element()).addClass(CHAT_MESSAGE_AVATAR_CLASS);
+
+    super._initMarkup();
+
+    const { name } = this.option();
+
+    const text = this._getAvatarInitials((name as any));
+
+    $('<div>')
+      .addClass(CHAT_MESSAGE_AVATAR_LETTERS_CLASS)
+      .text((text as any))
+      .appendTo(this.element());
+  }
+
+  _optionChanged(args: Record<string, unknown>): void {
+    const { name } = args;
+
+    switch (name) {
+      case 'text':
```

Comment:
Why we consider 'text' when I see that only option 'name' was declared for this component.

Reply:
[90dd3acd3c6dd8b06c21608a237d774ee8b6aa43]